### DISCUSSION TITLE
学堂路畔

### DISCUSSION THREAD

**Discussion by giscus** [ouuan/iles-blog#172] [Comments]
# 学堂路畔

在学堂路畔，与形形色色的路人擦肩而过，总能偶然听到一些有趣的只言片语；感觉可以记下来专门开一篇了（

https://ouuan.moe/post/2023/10/xuetang-roadside

**Comment by SunnyCloudYang** [comment]
现在是言也不行，行也不行（
ps. 欧老师终于有评论系统了，终于可以两边博客都嗑了（

**Reply by SunnyCloudYang** [reply]
草率了，原来是我一直没看到🥲

**Comment by junyu33** [comment]
其实还有比“十以内的加法”更困难的，那就是掌握 Peano 公理“后继”概念在不同自然语言体系的具体实现。作为一名合格的小学生，你得死记硬背：
- 阿拉伯数字 0 的后继是 1，1 的后继是 2，etc.
- 你至少得学会阿拉伯数字、中文汉字、英语这三种版本。
- 除了掌握他们的写法，你还需要掌握中文和英语中数字的读音。
- 为了把“后继”概念推广到 >9 的数，你还需要掌握十进制系统。

只要你掌握了这些，你就从理论上来说不用背而独立推导出这些口诀——只不过，死记硬背的条数从 45 条变成了 50 条。
